contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -353,6 +354,16 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5 {
         emit BurnConfigSet(_sanityRate, _weiToBurn);
     }
 
+    /// @dev Allow to set network proxy address by using burnConfigSetter
+    /// @param _newProxy new Kyber Proxy contract
+    function setNetworkProxy(IKyberProxy _newProxy) external onlyBurnConfigSetter {
+        require(_newProxy != IKyberProxy(0), "new proxy is 0");
+        if (_newProxy != networkProxy) {
+            emit KyberProxyUpdated(_newProxy, networkProxy);
+            networkProxy = _newProxy;
```

Comment:
Should we call update network contract here as well?

Reply:
Yes

Reply:
I Agree.
should update.

Reply:
Added update network